src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -512,9 +532,9 @@ def __call__(self, poller):
 class DeploymentOutputLongRunningOperation(LongRunningOperation):
     def __call__(self, result):
         from msrest.pipeline import ClientRawResponse
-        from msrestazure.azure_operation import AzureOperationPoller
+        from azure.cli.core.util import poller_classes
 
-        if isinstance(result, AzureOperationPoller):
+        if isinstance(result, poller_classes()):
```

Comment:
Is `get_poller_class` a better name?

Reply:
It could potentially be `get_poller_classes` since it's multiple classes I feel `poller_classes` also works especially in the form `isinstance(result, poller_classes())`.